config/enabled.yml

```diff
@@ -983,6 +983,11 @@ Performance/FlatMap:
   # This can be dangerous since `flat_map` will only flatten 1 level, and
   # `flatten` without any parameters can flatten multiple levels.
 
+Performance/InefficientHashSearch:
+  Description: 'Use `key?` or `value?` instead of `keys.include?` or `values.include?`'
+  Reference: 'https://github.com/JuanitoFatas/fast-ruby#hashkey-instead-of-hashkeysinclude-and-hashvalue-instead-of-hashvaluesinclude-code'
```

Comment:
This reference doesn't exist *yet*, but I have an outstanding PR which will create it: https://github.com/JuanitoFatas/fast-ruby/pull/154